File: expresença/ex1.c
#include <stdio.h>

int main() {
    char n[50];
    float sa, ns;
    float pa = 0.0;

    printf("Digite o nome do funcionário: ");
    scanf("%49s", n);
    printf("Digite o salário atual do funcionário: R$ ");
    scanf("%f", &sa);

    if (sa >= 0 && sa <= 400) {
        pa = 15;
    } else if (sa > 400 && sa <= 700) {
        pa = 12;
    } else if (sa > 700 && sa <= 1000) {
        pa = 10;
    } else if (sa > 1000 && sa <= 1800) {
        pa = 7;
    } else if (sa > 1800 && sa <= 2500) {
        pa = 4;
    } else if (sa > 2500) {
        pa = 0;
    }

    ns = sa + (sa * pa / 100);

    printf("\nNome do funcionário: %s\n", n);
    printf("Percentual de aumento: %.2f%%\n", pa);
    printf("Salário atual: R$ %.2f\n", sa);
    printf("Novo salário: R$ %.2f\n", ns);

    return 0;
}

File: expresença/ex1.py
n = input("Digite o nome do funcionário: ")
sa = float(input("Digite o salário atual do funcionário: R$ "))

if sa >= 0 and sa <= 400:
    pa = 15
elif sa > 400 and sa <= 700:
    pa = 12
elif sa > 700 and sa <= 1000:
    pa = 10
elif sa > 1000 and sa <= 1800:
    pa = 7
elif sa > 1800 and sa <= 2500:
    pa = 4
else:
    pa = 0

ns = sa + (sa * pa / 100)

print(f"\nNome do funcionário: {n}")
print(f"Percentual de aumento: {pa}%")
print(f"Salário atual: R$ {sa:.2f}")
print(f"Novo salário: R$ {ns:.2f}")
